src/components/Input/Input.component.tsx

```diff
@@ -40,44 +39,57 @@ const DisplayFieldSet = ({value, error, label}: InputProps) => (
  * <Input label='label' iconName='Globe'/>
  * ```
  */
-export const Input = (props: InputProps) => {
-    const {label, iconName, className = '', fullWidth, ...restProps} = props;
-    const {readOnly, error, required, value = ''} = restProps;
+export const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
+	const {
+		label,
+		iconName,
+		className = '',
+		fullWidth,
+		onClose,
+		searchQuery,
+		...restProps
+	} = props;
+	const {readOnly, error, required, value = ''} = restProps;
 
-    const inputLabel = `${label}${required ? ' *' : ''}`;
+	const inputLabel = `${label}${required ? ' *' : ''}`;
 
-    const togglePlaceholderValue = (event: React.ChangeEvent<HTMLInputElement>, type: string) => {
-        if (!readOnly) {
-            event.target.placeholder = type === 'focus' ? '' : inputLabel;
-        }
-    }
+	/**
+	 * Toggle placeholder value on type change
+	 * @param event - HTML input event
+	 * @param type - focus, blur
+	 */
+	const togglePlaceholderValue = (
+		event: React.ChangeEvent<HTMLInputElement>,
+		type: string
+	) => {
+		if (!readOnly) {
+			event.target.placeholder = type === 'focus' ? '' : inputLabel;
```

Comment:
```suggestion
			event.target.placeholder = (type === 'focus') ? '' : inputLabel;
```
Just because it makes the code more readable